Sources/VLCVideoOptionsControlBar.swift

```diff
@@ -0,0 +1,142 @@
+/*****************************************************************************
+ * VLCMediaViewEditCell.swift
+ *
+ * Copyright © 2018 VLC authors and VideoLAN
+ * Copyright © 2018 Videolabs
```

Comment:
Header naming should reflect the filename. 😅 
Additionally, bump the year and remove the Videolabs copyright.

Reply:
Haha copied that and forgot to change it 😅😂